src/hooks/useTokenInfoFromActiveList.ts

```diff
@@ -3,9 +3,7 @@ import { useWeb3React } from '@web3-react/core'
 import { useMemo } from 'react'
 import { useCombinedActiveList } from 'state/lists/hooks'
 
-/**
- * Returns a WrappedTokenInfo from the active token lists when possible,
- * or the passed token otherwise. */
+/** Returns a WrappedTokenInfo from the active token lists when possible, or the passed token otherwise. */
```

Comment:
Nit: this one could also be swapped with //

Reply:
This is a doc-comment - using `/**` ensures an IDE will surface this as documentation for the function.